ajax/libs/unveil2/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "unveil2",
+  "filename": "jquery.unveil2.min.js",
+  "description": "A very lightweight jQuery plugin to lazy load images",
+  "homepage": "https://nabble.github.io/unveil2/",
+  "author": "nabble",
+  "keywords": [
+    "lazyload",
+    "jquery"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nabble/unveil2.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nabble/unveil2.git",
+    "fileMap": [
+      {
+        "basePath": "",
```

Comment:
Let's use `dist` as basePath.
For more details, you can refer 
1. https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md
2. https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING-WIP.md#2-configure-auto-update